build.sbt

```diff
@@ -11,6 +11,15 @@ lazy val core =
           "Automatic-Module-Name" -> "akka.projection.core"))
     .settings(Protobuf.settings)
 
+lazy val coreTest =
```

Comment:
Should we consider moving all tests from `akka-projection-core` to this project?

Reply:
I had to move more tests over as part of #381

Reply:
yes, good to have all tests in `akka-projection-core-test` then

Reply:
Ok. TODO in #381.